openshift/sso72-x509.yaml

```diff
@@ -242,7 +242,7 @@ objects:
         - "rh-sso"
         from:
           kind: ImageStreamTag
-          name: "${NAME}:${IMAGE_TAG}"
+          name: "${NAME}:${VERSION}"
```

Comment:
`VERSION` is not intended to be used as tag. It is only for metadata/label of the version of RHSSO. Please undo this change.

Reply:
Will do Clecio. This change was made as I was still trying to use the OCP-SSO documentation to deploy; should have deleted this PR prior, apologies.